solr/prometheus-exporter/src/java/org/apache/solr/prometheus/exporter/SolrExporter.java

```diff
@@ -144,39 +157,68 @@ private SolrScraper createScraper(SolrScrapeConfiguration configuration, Prometh
             factory.createStandaloneSolrClient(configuration.getSolrHost().get()), requestExecutor);
       case CLOUD:
         return new SolrCloudScraper(
-            factory.createCloudSolrClient(configuration.getZookeeperConnectionString().get()), requestExecutor, factory);
+            factory.createCloudSolrClient(configuration.getZookeeperConnectionString().get()),
+            requestExecutor,
+            factory);
       default:
-        throw new RuntimeException(String.format(Locale.ROOT, "Invalid type: %s", configuration.getType()));
+        throw new RuntimeException(
+            String.format(Locale.ROOT, "Invalid type: %s", configuration.getType()));
```

Comment:
String concatenation would be shorter; probably wouldn't wrap

Reply:
Fixed in 6f59d6fa54dd1711174bd1e607b7c14e54fa6cd7